impeller/renderer/backend/vulkan/allocator_vk.cc

```diff
@@ -149,11 +228,25 @@ AllocatorVK::AllocatorVK(std::weak_ptr<Context> context,
     VALIDATION_LOG << "Could not create memory allocator";
     return;
   }
+
+  {
+    // Query texture support.
+    // TODO(jonahwilliams):
+    // https://github.com/flutter/flutter/issues/129784
+    physical_device.getMemoryProperties(&memory_properties_);
```

Comment:
Wasn't this already done above on line 182? Or is the scope significant somehow?

Reply:
oh good call. Will fix

Reply:
Done